charts/hivemq-operator/Chart.yaml

```diff
@@ -19,4 +19,4 @@ dependencies:
   - name: kube-prometheus-stack
     repository: https://prometheus-community.github.io/helm-charts
     condition: monitoring.dedicated, monitoring.enabled
-    version: 14.*
+    version: 30.*
```

Comment:
why did you choose 30 here?

Reply:
before 30, there was no compatibility with k8s v1.25 and version 40 did not work out of the box